README.md

```diff
@@ -30,6 +30,8 @@ We recommend using the Managed Config Controller service which comes bundles wit
 
 See the Google Cloud [quickstart](https://cloud.google.com/anthos-config-management/docs/tutorials/manage-resources-config-controller) guide for getting up and running with Config Controller
 
+A setup script is provided in the repository [gcp-tools](https://github.com/ssc-spc-ccoe-cei/gcp-tools/blob/main/scripts/bootstrap/setup-kcc.sh) is provided that will configure the Config Controller cluster.  The instructions in the [Advanced Install](https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/advanced-install.md) are automated as part of the [setup-kcc.sh](https://github.com/ssc-spc-ccoe-cei/gcp-tools/blob/main/scripts/bootstrap/setup-kcc.sh) script.
```

Comment:
For clarity I would remove the words `is provided` in this sentence:

A setup script is provided in the repository [gcp-tools](https://github.com/ssc-spc-ccoe-cei/gcp-tools/blob/main/scripts/bootstrap/setup-kcc.sh) that will configure the Config Controller cluster.